dev.linwood.butterfly.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id": "dev.linwood.butterfly",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "butterfly",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network"
+    ],
+    "modules": [
+        {
```

Comment:
because this is the main module, and the rest are dependencies, it should be last in the list. This is the common practice